src/Zynga/Framework/PgData/V1/PgRow.hh

```diff
@@ -52,4 +57,13 @@ abstract class PgRow extends StorableObject implements PgRowInterface {
     return $this->pgModel()->writer()->delete($this, $shouldUnlock);
   }
 
+  public function isTombstoned(): bool {
+    return $this->_ignore_tombstoned->get();
+  }
+
+  public function setTombstone(): bool {
+    $this->_ignore_tombstoned->set(true);
+    return $this->_ignore_tombstoned->get();
+  }
```

Comment:
nitpick: may be we should do it in traditional setter way with void return type and use isTombstoned() for any checks.

Reply:
I would consider renaming the method to be something like `tombstoneRow()`, instead of calling it `setTombstone()`. Because it would make it clearer what the method is responsible for. Since seeing set I would assume it could enable or disable the tombstone()